firefox-ios/Client/Frontend/Browser/RecordVisitObservationManager.swift

```diff
@@ -14,7 +14,8 @@ protocol RecordVisitObserving: AnyObject {
     func resetRecording()
 }
 
-class RecordVisitObservationManager: RecordVisitObserving {
+// TODO: FXIOS-TODO: RecordVisitObservationManager should not be @unchecked Sendable
+class RecordVisitObservationManager: RecordVisitObserving, @unchecked Sendable {
```

Comment:
I think this is a new warning, the count increased (?!)

<img width="1332" height="401" alt="Screenshot 2025-12-03 at 2 22 22 PM" src="https://github.com/user-attachments/assets/525f9a53-697f-4480-be72-fb37616afa75" />

Not sure why I have this now